From 55957faf1272c8f5f304909faeebf647a78e3701 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 9 Sep 2015 07:19:45 +0000
Subject: [PATCH] console: Fix C syntax errors for function declaration

To address this, the semicolons after the function parameters should be
replaced by commas, and the last one should be omitted

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
Upstream-Status: Pending

 src/efi/console.c | 26 +++++++++++++-------------
 1 file changed, 13 insertions(+), 13 deletions(-)

diff --git a/src/efi/console.c b/src/efi/console.c
index 6206c80..66aa88f 100644
--- a/src/efi/console.c
+++ b/src/efi/console.c
@@ -27,8 +27,8 @@
 struct _EFI_SIMPLE_TEXT_INPUT_EX_PROTOCOL;
 
 typedef EFI_STATUS (EFIAPI *EFI_INPUT_RESET_EX)(
-        struct _EFI_SIMPLE_TEXT_INPUT_EX_PROTOCOL *This;
-        BOOLEAN ExtendedVerification;
+        struct _EFI_SIMPLE_TEXT_INPUT_EX_PROTOCOL *This,
+        BOOLEAN ExtendedVerification
 );
 
 typedef UINT8 EFI_KEY_TOGGLE_STATE;
@@ -44,29 +44,29 @@ typedef struct {
 } EFI_KEY_DATA;
 
 typedef EFI_STATUS (EFIAPI *EFI_INPUT_READ_KEY_EX)(
-        struct _EFI_SIMPLE_TEXT_INPUT_EX_PROTOCOL *This;
-        EFI_KEY_DATA *KeyData;
+        struct _EFI_SIMPLE_TEXT_INPUT_EX_PROTOCOL *This,
+        EFI_KEY_DATA *KeyData
 );
 
 typedef EFI_STATUS (EFIAPI *EFI_SET_STATE)(
-        struct _EFI_SIMPLE_TEXT_INPUT_EX_PROTOCOL *This;
-        EFI_KEY_TOGGLE_STATE *KeyToggleState;
+        struct _EFI_SIMPLE_TEXT_INPUT_EX_PROTOCOL *This,
+        EFI_KEY_TOGGLE_STATE *KeyToggleState
 );
 
 typedef EFI_STATUS (EFIAPI *EFI_KEY_NOTIFY_FUNCTION)(
-        EFI_KEY_DATA *KeyData;
+        EFI_KEY_DATA *KeyData
 );
 
 typedef EFI_STATUS (EFIAPI *EFI_REGISTER_KEYSTROKE_NOTIFY)(
-        struct _EFI_SIMPLE_TEXT_INPUT_EX_PROTOCOL *This;
-        EFI_KEY_DATA KeyData;
-        EFI_KEY_NOTIFY_FUNCTION KeyNotificationFunction;
-        VOID **NotifyHandle;
+        struct _EFI_SIMPLE_TEXT_INPUT_EX_PROTOCOL *This,
+        EFI_KEY_DATA KeyData,
+        EFI_KEY_NOTIFY_FUNCTION KeyNotificationFunction,
+        VOID **NotifyHandle
 );
 
 typedef EFI_STATUS (EFIAPI *EFI_UNREGISTER_KEYSTROKE_NOTIFY)(
-        struct _EFI_SIMPLE_TEXT_INPUT_EX_PROTOCOL *This;
-        VOID *NotificationHandle;
+        struct _EFI_SIMPLE_TEXT_INPUT_EX_PROTOCOL *This,
+        VOID *NotificationHandle
 );
 
 typedef struct _EFI_SIMPLE_TEXT_INPUT_EX_PROTOCOL {
-- 
2.5.1

